CHANGELOG.md

```diff
@@ -69,6 +70,27 @@ All notable changes to "Azure Toolkit for IntelliJ IDEA" will be documented in t
   - [3.0.7](#307)
   - [3.0.6](#306)
 
+## 3.55.0
+### Added
+- New Azure Resource Connector explorer for connection management
```

Comment:
add below sub lines for connector explorer: ?
Add to connect a new connection.
Delete a existing connection.
Edit a existing connection.
Navigate a existing connection to its properties view.